scalafix-core/shared/src/main/scala/scalafix/internal/config/ScalafixConfig.scala

```diff
@@ -18,7 +20,8 @@ case class ScalafixConfig(
     reporter: ScalafixReporter = ScalafixReporter.default,
     patches: ConfigRulePatches = ConfigRulePatches.default,
     dialect: Dialect = ScalafixConfig.DefaultDialect,
-    lint: LintConfig = LintConfig.default
+    lint: LintConfig = LintConfig.default,
+    diffs: Option[List[GitDiff]] = None
```

Comment:
Is this configurable via .scalafix.conf? Is there a way we can keep GitDiff out RuleCtx/ScalafixConfig?

Reply:
Nope only via cli, I guess I should add it to the config file as well.

Reply:
I think we can keep this out of ScalafixConfig with the suggestion I made to use a different Rule.apply in CliRunner.